gkeys/gkeys/base.py

```diff
@@ -301,9 +305,12 @@ def setup(self, args, configs):
             args = self.parse_args(args)
         if args.config:
             self.config.defaults['config'] = args.config
+            self.config.defaults['configdir'] = os.path.dirname(args.config)
             self.config.read_config()
         else:
             self.config.read_config(configs)
+        if args.email in ['expiry']:
+            self.config.read_config(os.path.abspath(os.path.join(self.config.defaults['configdir'], "email.conf")))
```

Comment:
this is not using stacking.  In this case it is ignoring the real config file and only using the defaults plus the email.conf file.

You want to pass the config parser all files in the order you want them stacked and it will do it all for you.